qcodes/instrument_drivers/rohde_schwarz/SGS100A.py

```diff
@@ -2,6 +2,7 @@
 
 
 class RohdeSchwarz_SGS100A(VisaInstrument):
+
```

Comment:
We don't use empty lines  here or after the docstring, your auto pep8 is probably doing it for you @eendebakpt  had the same issue. 

Also since you are here why not switch to proper triple double quotes ?  :heart: